lib/aws/rails/mailer.rb

```diff
@@ -1,21 +1,21 @@
+# frozen_string_literal: true
+
 require 'aws-sdk-ses'
 
 module Aws
   module Rails
-
     # Provides a delivery method for ActionMailer that uses Amazon Simple Email
     # Service.
-    # 
+    #
     # Once you have an SES delivery method you can configure Rails to
     # use this for ActionMailer in your environment configuration
     # (e.g. RAILS_ROOT/config/environments/production.rb)
     #
-    #     config.action_mailer.delivery_method = :aws_sdk
+    #     config.action_mailer.delivery_method = :ses
     #
-    # Uses the AWS SDK for Ruby V2's credential provider chain when creating an
+    # Uses the AWS SDK for Ruby V3's credential provider chain when creating an
```

Comment:
Maybe just remove the "V3's" altogether, so that it can't fall out of date in the future?

Reply:
Good call. I am going to use language such as 'latest version'.